packages/kbn-test/src/functional_test_runner/lib/config/schema.ts

```diff
@@ -200,7 +201,7 @@ export const schema = Joi.object()
       .keys({
         license: Joi.valid('basic', 'trial', 'gold').default('basic'),
         from: Joi.string().default('snapshot'),
-        serverArgs: Joi.array().items(Joi.string()),
+        serverArgs: Joi.array().items(Joi.string()).default([]),
```

Comment:
This will help us to clean code a bit and avoid `config.get('esTestCluster.serverArgs') || []` in many places